src/modules/auth/auth.service.test.ts

```diff
@@ -7,36 +7,39 @@ import { AuthService } from './auth.service';
 import { ApiKeyStrategy } from './strategy/api-key.strategy';
 
 describe('AuthService', () => {
-  let spyService: AuthService;
+  let authsService: AuthService;
```

Comment:
very small nitpick, but should this be `authService` instead of `authsService`?

Reply:
this is updated now